src/main/java/org/jabref/gui/collab/metedatachange/MetadataChange.java

```diff
@@ -0,0 +1,32 @@
+package org.jabref.gui.collab.metedatachange;
+
+import org.jabref.gui.collab.ExternalChange;
+import org.jabref.gui.collab.ExternalChangeResolverFactory;
+import org.jabref.gui.undo.NamedCompound;
+import org.jabref.logic.bibtex.comparator.MetaDataDiff;
+import org.jabref.logic.l10n.Localization;
+import org.jabref.model.database.BibDatabaseContext;
+
+public final class MetadataChange extends ExternalChange {
+    private final MetaDataDiff metaDataDiff;
+
+    public MetadataChange(MetaDataDiff metaDataDiff, BibDatabaseContext databaseContext, ExternalChangeResolverFactory externalChangeResolverFactory) {
+        super(databaseContext, externalChangeResolverFactory);
+        this.metaDataDiff = metaDataDiff;
+        setChangeName(Localization.lang("Metadata change"));
+    }
+
+    @Override
+    public void applyChange(NamedCompound undoEdit) {
+        // TODO: Metadata edit should be undoable
```

Comment:
Will you do the ToDo or is this postponed?

Reply:
Couldn't find a `UndoableMetadaEdit` class. So I thought maybe Metadata is not meant to be undoable.

Reply:
As discussed last time, ignore the missing metatata cahnges for the moment.